src/user/user.service.ts

```diff
@@ -1,17 +1,18 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 
-import { Repository } from 'typeorm';
 import { hash } from 'bcrypt';
 
+import { UserRepository } from './user.repository';
+
 import { User } from './entities/user.entity';
 import { AddUserInput, AddUserOutput } from './dto/add-user.dto';
 
 @Injectable()
 export class UserService {
   constructor(
     @InjectRepository(User)
-    private userRepository: Repository<User>,
+    private userRepository: UserRepository,
```

Comment:
@yashmurty @war1oc When we are injecting a custom repository, we don't need to annotate it separately using `@InjectRepository(User)`.  It is only needed for dynamic repository injecting (I guess).

Reply:
@asad-mlbd  you are spot on. I fixed it now. :pray: